README.md

```diff
@@ -10,4 +10,4 @@ go build -o signoz-troubleshoot .
 ```
  ./signoz-troubleshoot checkEndpoint --endpoint=<endpoint-to-check>
  ```
- Eg. `./signoz-troubleshoot checkEndpoint --endpoint=http://localhost:4317
+ Eg. `./signoz-troubleshoot checkEndpoint --endpoint=localhost:4317
```

Comment:
Is this tested without `http://`, I remember trying without it and signoz was not receiving data

Reply:
yes, it is tested.